src/app.ts

```diff
@@ -1,42 +1,60 @@
-import path from 'path';
-import favicon from 'serve-favicon';
-import compress from 'compression';
-import helmet from 'helmet';
-import cors from 'cors';
-
-import feathers from '@feathersjs/feathers';
-import configuration from '@feathersjs/configuration';
-import express from '@feathersjs/express';
-import socketio from '@feathersjs/socketio';
-
-
-import { Application } from './declarations';
-import logger from './logger';
-import middleware from './middleware';
-import services from './services';
-import appHooks from './app.hooks';
-import channels from './channels';
-import { HookContext as FeathersHookContext } from '@feathersjs/feathers';
-import authentication from './authentication';
-import mongoose from './mongoose';
+import path from "path";
+import favicon from "serve-favicon";
+import compress from "compression";
+import helmet from "helmet";
+import cors from "cors";
+
+import feathers from "@feathersjs/feathers";
+import configuration from "@feathersjs/configuration";
+import express from "@feathersjs/express";
+import socketio from "@feathersjs/socketio";
+import swagger from "feathers-swagger";
+
+import { Application } from "./declarations";
+import logger from "./logger";
+import middleware from "./middleware";
+import services from "./services";
+import appHooks from "./app.hooks";
+import channels from "./channels";
+import { HookContext as FeathersHookContext } from "@feathersjs/feathers";
+import authentication from "./authentication";
+import mongoose from "./mongoose";
+
 // Don't remove this comment. It's needed to format import lines nicely.
 
 const app: Application = express(feathers());
-export type HookContext<T = any> = { app: Application } & FeathersHookContext<T>;
+export type HookContext<T = any> = {
+  app: Application;
+} & FeathersHookContext<T>;
 
 // Load app configuration
 app.configure(configuration());
 // Enable security, CORS, compression, favicon and body parsing
-app.use(helmet({
-  contentSecurityPolicy: false
-}));
+app.use(
+  helmet({
+    contentSecurityPolicy: false,
+  })
+);
 app.use(cors());
 app.use(compress());
 app.use(express.json());
 app.use(express.urlencoded({ extended: true }));
-app.use(favicon(path.join(app.get('public'), 'favicon.ico')));
-// Host the public folder
-app.use('/', express.static(app.get('public')));
+app.use(favicon(path.join(app.get("public"), "favicon.ico")));
+
+// Configure Swagger API
+app.configure(
+  swagger({
+    docsPath: "/swagger",
+    uiIndex: true,
+    specs: {
+      info: {
+        title: "A test",
```

Comment:
how about the repo's name for now?
```suggestion
        title: "EddieCommunityApi",
```